app/access.py

```diff
@@ -34,6 +34,8 @@ def wrapper(self, *args, **kwargs):
             if self.current_user is not None:
                 if not self.current_user.is_active():
                     raise tornado.web.HTTPError(403, "User account expired")
+            else:
+                raise tornado.web.HTTPError(403, "Credentials missing or malformed")
```

Comment:
This should be a 401. Can we provide more explicit instructions to the user?  Something like: `Credentials malformed; expected header of form "authorization: token abc123"`

Reply:
We could probably catch this more explicitly on line 24 (`token_header.replace`)?